### PR TITLE
Develop - navbar and hero section

### PR DESCRIPTION
**Pull request by danielc-moveo**
both features are combined since the Searchbox component is common.

### CODE REVIEW

File: src/components/layout/navbar/Navbar.tsx [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
 const Navbar = () => {
-    return (
-        <Nav>
-            <div>first group</div>
-            <div>second group</div>
-        </Nav>
-    )
-}
+  const handleIconClick = (path: string) => {
+    console.log(path);
```
Comment (Aslanchik-m) [inline src/components/layout/navbar/Navbar.tsx]: Before pushing code to production remove all redundant console logs.

File: src/components/layout/footer/Styles.tsx [hunk 2]
```diff
@@ -13,11 +13,6 @@ export const FooterLink = styled(Link)`
   margin: 0 45px;
   text-decoration: none;
   color: ${colors.black};
-  //   font-family: HelveticaNeue;
   font-size: 18px;
```
Comment (Aslanchik-m) [inline src/components/layout/footer/Styles.tsx]: Avoid using pixels when working with font related properties, use rem or em.
Reference - 
[Pixels are dead](https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9)

File: src/components/layout/homepage/mobile-links/Styles.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+export const LinksContainer = styled.div`
+  display: flex;
+`;
+
+export const Link = styled(Link_)`
+  margin: 0 5px;
+  text-decoration: none;
+  color: ${colors.black};
+  padding: 10px;
+  background-color: ${colors.sand};
```
Comment (Aslanchik-m) [inline src/components/layout/homepage/mobile-links/Styles.tsx]: Great job exporting your colors from a global color file, but try naming them something a bit more descriptive than sand. Usually you'll have a primary color palette and you'll name them "primary-light" "primary-dark" "primary-orange" "primary-dark-orange" and so on.

File: src/components/layout/homepage/mobile-links/Styles.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  font-size: 13px;
+  font-weight: normal;
```
Comment (Aslanchik-m) [inline src/components/layout/homepage/mobile-links/Styles.tsx]: Avoid unnecessary properties, `font-weight: normal` is the default.

File: src/index.css [hunk 5]
```diff
@@ -1,13 +1,21 @@
...
 }
 
+@font-face {
+  font-family: HelveticaNeue;
+  src: local(HelveticaNeue-Thin),
+    url(./utils/fonts/HelveticaNeue-Thin.otf) format("otf");
+  font-style: normal;
+  font-stretch: normal;
+}
+
```
Comment (Aslanchik-m) [inline src/index.css]: Nice job

**Comment by Aslanchik-m** [comment]
Great job so far! 
Keep it up.

File: src/components/layout/footer/Footer.tsx [hunk 6]
```diff
@@ -6,12 +6,13 @@ const footerItems = [
...
 
-
 const Footer = () => {
   return (
     <Wrapper>
       {footerItems.map(({ title, path }, i) => (
-        <FooterLink to={path}>{title}</FooterLink>
+        <FooterLink key={i} to={path}>
+          {title}
+        </FooterLink>
```
Comment (Niv-hls) [inline src/components/layout/footer/Footer.tsx:15]: Good Change!
Always use key props when you rendering elements from an array (like here sith map function)

File: src/components/layout/footer/Styles.tsx [hunk 7]
```diff
@@ -12,12 +12,7 @@ export const Wrapper = styled.div`
...
   text-decoration: none;
-  color: ${colors.black};
-  //   font-family: HelveticaNeue;
-  font-size: 18px;
-  font-weight: 100;
-  font-stretch: normal;
-  font-style: normal;
-  line-height: normal;
-  letter-spacing: 1.13px;
+  color: ${colors.primary_black};
```
Comment (Niv-hls) [inline src/components/layout/footer/Styles.tsx:15]: Good work using global color!

File: src/components/layout/homepage/search-section/Styles.tsx [hunk 8]
```diff
@@ -1,26 +1,65 @@
...
+
+    & span {
+      font-size: 16px;
+      margin: 18px 0 13px 0;
+    }
+
+    & input {
+      font-size: 13px;
+    }
+  }
```
Comment (Niv-hls) [inline src/components/layout/homepage/search-section/Styles.tsx]: Good job using Media queries and nested Sass

File: src/components/layout/navbar/Navbar.tsx [hunk 9]
```diff
@@ -1,19 +1,47 @@
...
+        <SearchBox isNavbar={true} />
+        {rightGroupIcons.map(({ src, path, styles }, i) => (
+          <RightGroupIcon
+            isMainLogo={false}
+            alt=""
+            src={src}
+            onClick={() => handleRedirect(path)}
+            key={i}
+            styles={styles}
+          />
```
Comment (Niv-hls) [inline src/components/layout/navbar/Navbar.tsx:40]: Good implementations of Flexbox CSS

File: src/components/layout/navbar/Navbar.tsx [hunk 10]
```diff
@@ -1,19 +1,47 @@
...
+        <Logo src={logo} onClick={() => handleRedirect("/")} alt="" />
+        {leftGroupLinks.map(({ title, styles, path }, i) => (
+          <NavLink key={i} styles={styles} to={path}>
+            {title}
+          </NavLink>
+        ))}
+      </LeftGroup>
+      <RightGroup>
+        <SearchBox isNavbar={true} />
+        {rightGroupIcons.map(({ src, path, styles }, i) => (
```
Comment (Niv-hls) [inline src/components/layout/navbar/Navbar.tsx:32]: Nice way to pass props through components.

File: src/components/utils/common/CommonStyles.tsx [hunk 11]
```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import { DescriptionStyle } from "./interface";
+
+export const colors = {
+  primary_black: "black",
+  primary_yellow: "rgba(232,196,122,0.8)",
+  primary_grey: "#fafafa",
+};
+
+export const Description = styled.span<DescriptionStyle>`
```
Comment (Niv-hls) [inline src/components/utils/common/CommonStyles.tsx:10]: Nice job
always use interfaces to pass props into styled-components,
It helps to the team to keep tracking the code

**Pull request merged**